invenio_app_ils/ui/src/invenio_app_ils/forms/components/AccordionField/AccordionField.js

```diff
@@ -0,0 +1,85 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { Field } from 'formik';
+import { Accordion, Form, Icon, Segment } from 'semantic-ui-react';
+import './AccordionField.scss';
+
+export class AccordionField extends Component {
+  constructor(props) {
+    super(props);
+    this.fieldPath = props.fieldPath;
+    this.label = props.label;
+    this.required = props.required;
+    this.uiProps = props.uiProps;
```

Comment:
Are all these needed to be assigned to `AccordionField`? they can be used directly

Reply:
If by needed you mean it's not gonna work otherwise, then no it's not needed :P I like the approach of having the props that a resuable component can accept in the constructor, as long as the component is not depending on these props for update ofc. The `uiProps` is there just for an extra hook to override semantic related props.